ddprof-lib/src/main/cpp/flightRecorder.cpp

```diff
@@ -322,6 +323,14 @@ off_t Recording::finishChunk(bool end_recording) {
         writeRecordingInfo(_buf);
     }
 
+    // this will not include counters for any counters updated during writing the constant pool
```

Comment:
I think there is a typo here

```suggestion
    // this will not include counts for any counters updated during writing the constant pool
```